### PR TITLE
Add information for CRI-O and containerd log parsing

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
Add containerd Fluent Bit Parser
Remove quetes for crio Parser to remove time parsing errors

Related to https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1418 https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1417 https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1414

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/ContainerdLogs.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+{
+   "timestamp":1613653807337,
+   "stream":"stdout",
+   "logtag":"F",
+   "log":"this is a log message"
+}
+```
+
+In order to have time from log line as a string value in log in Sumo please
+remove following lines from `containerd` parser configuration:
```
Comment (pmalek-sumo) [inline deploy/docs/ContainerdLogs.md:61]:
```suggestion
remove the following lines from `containerd` parser configuration:
```

File: deploy/docs/ContainerdLogs.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Configuration for containerd log format
+
+In order to collect logs in CRI-O log format modify Fluent Bit input section to following form:
```
[the remark this answers is not in the document]
Comment (sumo-drosiek) [reply deploy/docs/ContainerdLogs.md:3]: We could say even CRI-containerd

**Comment by sumo-drosiek** [comment]
Should we have similar document for docker?

File: deploy/docs/ContainerdLogs.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kkujawa-sumo) [reply deploy/docs/ContainerdLogs.md:3]: I think that is easier for user to have separate sections for CRIO and containerd, time format is different so different parser needs to be used.

**Comment by kkujawa-sumo** [comment]
> Should we have similar document for docker?

I think that it is good idea. I'll prepare such instruction and then we could have one document "ContainerLogs.md" for docker, CRI-O and containerd.

File: deploy/docs/ContainerdLogs.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sumo-drosiek) [reply deploy/docs/ContainerdLogs.md]: I mean the format is sometimes referred as `CRI-Containerd`

File: deploy/docs/ContainerLogs.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+   "time":"2021-02-18T12:14:16.722375295+00:00",
+   "stream":"stdout",
+   "logtag":"F",
+   "log":"this is a log message"
+}
+```
+
+## Configuration for containerd log format
+
+In order to collect logs in containerd log format modify, Fluent Bit input section to following form:
```
Comment (astencel-sumo) [inline deploy/docs/ContainerLogs.md:145]:
```suggestion
In order to collect logs in containerd log format, modify Fluent Bit input section to following form:
```

File: deploy/docs/ContainerLogs.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
+# Container log parsing
```
Comment (astencel-sumo) [inline deploy/docs/ContainerLogs.md:1]: This new documentation article is only linked in a comment in Fluent Bit configuration. Shouldn't we also link to it from some other doc (`.md` file)?

Comment (kkujawa-sumo) [reply deploy/docs/ContainerLogs.md:1]: I added link to this document in [values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1441/files#diff-da997ab241c9344f2bb085822fe04e4e43eaf0f8034a9d15dc3942b8a5fbe046R850)

**Review by astencel-sumo — APPROVED** [review]

File: deploy/helm/sumologic/README.md [hunk 5]
```diff
@@ -172,7 +172,7 @@ Parameter | Description | Default
 `fluent-bit.resources` | Resources for Fluent-bit daemonsets. | `{}`
 `fluent-bit.enabled` | Flag to control deploying Fluent-bit Helm sub-chart. | `true`
 `fluent-bit.config.service` | Configure Fluent-bit Helm sub-chart service. | [fluent-bit.config.service in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L817-L827)
-`fluent-bit.config.inputs` | Configure Fluent-bit Helm sub-chart inputs. | [fluent-bit.config.inputs in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L828-L895)
+`fluent-bit.config.inputs` | Configure Fluent-bit Helm sub-chart inputs. Configuration for logs from different container runtimes describe in [Container log parsing](../../docs/ContainerLogs.md). | [fluent-bit.config.inputs in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L828-L895)
```
Comment (astencel-sumo) [inline deploy/helm/sumologic/README.md:175]:
```suggestion
`fluent-bit.config.inputs` | Configure Fluent-bit Helm sub-chart inputs. Configuration for logs from different container runtimes is described in [Container log parsing](../../docs/ContainerLogs.md). | [fluent-bit.config.inputs in values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.0.0/deploy/helm/sumologic/values.yaml#L828-L895)
```

**Pull request merged**